src/syscall_mock/mod.rs

```diff
@@ -20,7 +19,7 @@ pub struct SyscallMock {
     context: Context,
     tracee_pid: Pid,
     protocol: Protocol,
-    unmocked_commands: Vec<Vec<u8>>,
+    unmocked_commands: Vec<PathBuf>,
```

Comment:
I could change this type from `PathBuf` back to `Vec<u8>`. That would remove the need to make `Comand::path_from_bytes` a public method. On the other hand `PathBuf` or `Path` seem like the "correct" type for an OS command.

Reply:
I changed the type of `unmocked_commands` back to `Vec<Vec<u8>>`.